setup.cfg

```diff
@@ -60,6 +60,8 @@ install_requires =
     certifi>=2017.4.17
     typing_extensions>=4.3,<5
     filelock>=3.5,<4
+    platformdirs
+    tomlkit
```

Comment:
Do we need to get legal to review when we add new dependencies?

Reply:
Sure, do you have description, or example of what to do?

Reply:
The Python docs suggested it https://docs.python.org/3/library/tomllib.html
Since we support 3.7-3.9. We can't assume that the standard lib one is available